kardia/params.go

```diff
@@ -47,3 +51,152 @@ func (ec *Client) GetMaxProposers(ctx context.Context) (int64, error) {
 	}
 	return result.MaxProposers.Int64(), nil
 }
+
+// GetParams returns value of params by indexes
+func (ec *Client) GetParams(ctx context.Context) (map[string]string, error) {
+	params := make(map[string]string)
+	for i, paramName := range cfg.ParamKeys {
+		payload, err := ec.paramsUtil.Abi.Pack("getParam", uint8(i))
+		if err != nil {
+			return nil, err
+		}
+		res, err := ec.KardiaCall(ctx, contructCallArgs(ec.paramsUtil.ContractAddress.Hex(), payload))
+		if err != nil {
+			return nil, err
+		}
+
+		var result struct {
+			Value *big.Int
+		}
+		// unpack result
+		err = ec.paramsUtil.Abi.UnpackIntoInterface(&result, "getParam", res)
+		if err != nil {
+			ec.lgr.Error("Error unpacking params", zap.Int("id", i), zap.String("name", paramName), zap.Error(err))
+			return nil, err
+		}
+		params[cfg.ParamKeys[i]] = result.Value.String()
+	}
+	return params, nil
+}
+
+// GetProposalDetails returns detail of a proposal by ID
+func (ec *Client) GetProposalDetails(ctx context.Context, proposalId *big.Int) (*types.ProposalDetail, error) {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>method parameter `proposalId` should be `proposalID` (golint)